### PR TITLE
Judges | Assign cases | Assign to 'Other'

### PR DESCRIPTION
**Pull request by evankroske**
- Fixes https://github.com/department-of-veterans-affairs/caseflow/issues/5174

### Description
Allows judges to assign cases to any attorney at the board.

Screen shot of assign widget when using it to assign to an attorney not on the user's team

### Testing Plan
- [x] Become BVARERDMAN.
- [x] Assign a case to Lela Shields.
- [x] Become BVAAABSHIRE.
- [x] Confirm that the case is assigned to Lela.
- [x] On "Cases to Assign", assign an unassigned case to Jazlyn Schimmel.
- [x] Become BVARERDMAN.
- [x] Confirm that the case has been assigned to Jazlyn.

### CODE REVIEW

File: client/app/queue/reducers.js [hunk 1]
```diff
@@ -325,6 +326,29 @@ const workQueueReducer = (state = initialState, action = {}) => {
     const appealId = action.payload.task.id;
     const appeal = state.tasksAndAppealsOfAttorney[action.payload.previousAssigneeId].data.appeals[appealId];
 
+    const tasksAndAppealsOfAssignee = update(
+      state.tasksAndAppealsOfAttorney[action.payload.assigneeId] ||
```
Comment (mdbenjam) [inline client/app/queue/reducers.js]: Why is this change necessary?

Comment (evankroske) [reply client/app/queue/reducers.js]: Otherwise, when the reducer tries to add the task to the assignee's list of tasks, it throws an exception. The old code assumed that tasksAndAppealsOfAttorney[assigneeId] was defined, which isn't true when the assignee isn't on the assigner's team.
